packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js

```diff
@@ -256,10 +256,17 @@ Image.propTypes = {
   }).isRequired,
 };
 
+// This styled component is used to make sure the tooltip is above the popover
```

Comment:
it's nicer to go straight to the point in your comments
```suggestion
// Make sure the tooltip is above the popover
```